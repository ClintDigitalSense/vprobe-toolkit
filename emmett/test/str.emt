string garth;

Probe0 {
	garth = "fooble";
	strAggr[garth, "warb"] <- 123;
        garth = "woobers";
}
