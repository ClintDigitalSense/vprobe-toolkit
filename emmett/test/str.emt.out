(version 0.2)
(defstring garth)
(defaggr strAggr 0 2)

 ;;;;;
(vprobe Probe0 ( do  ( setstr garth   "fooble")( aggr  strAggr () ( garth "warb") 123)( setstr garth   "woobers")))